lib/api/apiUtils/object/versioning.js

```diff
@@ -55,14 +55,46 @@ function _storeNullVersionMD(bucketName, objKey, objMD, options, log, cb) {
     });
 }
 
+function _getVersionLocation(bucketName, objKey, options, log, cb) {
```

Comment:
please add jsdoc.  do you think this is better called, getDeleteLocations?  Is this used for anything else?